cime_config/machines/config_machines.xml

```diff
@@ -3133,6 +3133,114 @@
     </environment_variables>
   </machine>
 
+  <machine MACH="chicoma-cpu">
+    <DESC>Chicoma CPU-only nodes at LANL IC. Each node has 2 AMD EPYC 7H12 64-Core (Milan) 512GB</DESC>
+    <NODENAME_REGEX>ch-fe.*.lanl.gov</NODENAME_REGEX>
+    <OS>Linux</OS>
+    <COMPILERS>gnu,nvidia,aocc,amdclang</COMPILERS>
+    <MPILIBS>mpich</MPILIBS>
+    <PROJECT>t22_e3sm-gpu</PROJECT>
```

Comment:
Project name is actually
t22_e3sm-gpu_g

Reply:
It sounds like we need to use `t22_ocean_time_step` or something else for the CPU side.  Then, `t22_e3sm-gpu_g` when we do `chicoma-gpu`.

Reply:
Do we want to leave PROJECT unset for this PR? Some machines, like theta, require a PROJECT but do not set one by default

Reply:
That might be the best solution, rather than trying to create an inclusive IC project with no resources.

Reply:
Yes, I agree.